### DISCUSSION TITLE
Does Redash.io works with Bubble.io

### DISCUSSION THREAD

**Discussion by RenatoAsent** [getredash/redash#6392] [Q&A]
I know that bubble.io servers operate in AWS Cloud, but i could not find a way to connect my database to redash.io.

Does anyone know if this integration is possible? I would like to use redash.io for my company.

**Comment by justinclift** [comment]
We don't *yet* have a capability for interacting with Bubble.io.

It would be possible for someone with Python experience to add that capability though. :smile:

When looking at the list of available Python libraries, there is already one for Bubble.io:

https://pypi.org/project/bubble-client/

So, that means it shouldn't be "too hard" for someone with Python experience to make it happen.

If you know someone that could add the capability, then please feel welcome to point them at the above "Bubble client" project and Redash.  We'd welcome their contribution. :smile:
